infrastructure/ci/static-analysis-if-changed.sh

```
@@ -1,17 +1,15 @@
 #!/usr/bin/env bash
 set -ex
```

Okay, `set -ex` will help fail fast. I should have spotted that.

I'm still not sure we need to run `bundle install` or `yarn install` though.

All deps _should_ be installed and in `Gemfile.lock` and `packages.json` when we push changes.

---

Good point. I forgot that that is part of the build process which runs before this script.